package-lock.json

```diff
@@ -2640,17 +2640,6 @@
         "humanize-ms": "^1.2.1"
       }
     },
-    "ajv": {
-      "version": "6.1.1",
-      "resolved": "https://registry.npmjs.org/ajv/-/ajv-6.1.1.tgz",
-      "integrity": "sha1-l41Zf7wrfQ5aXD3esUmmgvKr+g4=",
-      "dev": true,
-      "requires": {
-        "fast-deep-equal": "^1.0.0",
-        "fast-json-stable-stringify": "^2.0.0",
-        "json-schema-traverse": "^0.3.0"
-      }
-    },
     "ajv-errors": {
```

Comment:
dumb q: why is ajv-errors still included at this level is ajv isn't?

Reply:
Sorry I missed this comment in the files view as the lock file gets collapsed.
Think this could be the reason for https://github.com/forcedotcom/salesforcedx-vscode/pull/3942/files